.github/workflows/publish_container.yml

```
@@ -4,28 +4,52 @@ on:
   push:
     branches:
       - main
+    tags:
+      - 'v*'
 
 jobs:
```

Since the build and push steps are the same except for the tags, we could do something like this to set the desired tags to an environment variable. I'm not sure this is simpler though. If you don't like this, we could also consider just splitting the workflow into two separate workflows.
```suggestion
jobs:
  image-tag-latest:
    if: github.ref == 'refs/heads/main'
    run: |
      echo "IMAGE_TAGS=ghcr.io/${{ github.repository }}/hushline:latest" >> $GITHUB_ENV
      
  image-tag-release:
    if: startsWith(github.ref, 'refs/tags/v')
    run: |
      TAGS="ghcr.io/${{ github.repository }}/hushline:${{ github.ref_name }} \n
                   ghcr.io/${{ github.repository }}/hushline:release"
      echo "IMAGE_TAGS<<EOF" >> $GITHUB_ENV
      echo $TAGS >> $GITHUB_ENV
      echo "EOF" >> $GITHUB_ENV
```